details/cheatsheets.md

```diff
@@ -2,7 +2,6 @@
 
   - All-in-one cheatsheets:
     + [💡  Quick Reference Sheet 💡](https://www.linkedin.com/posts/asif-bhat_quick-reference-sheet-ml-dl-ai-activity-6640966422036078592-BB4l)
-    + [All  📝Cheat Sheets📝 in one place](https://github.com/abhat222/Data-Science--Cheat-Sheet)
```

Comment:
Instead of taking it down, we could instead add a `[Deadlink]` label next to it.